docs/user_guide/selection/DropHighPSIFeatures.rst

```diff
@@ -68,6 +68,21 @@ to the PSI value. The most commonly used thresholds are:
 - Below 10%, the variable has not experienced a significant shift.
 - Above 25%, the variable has experienced a major shift.
 - Between those two values, the shift is intermediate.
+- 'auto' threshold will be calculated based on size of dataset and number of bins.
```

Comment:
Could we modify slightly to

'auto': the threshold will be calculated based on the size of the base and target datasets and the number of bins.